src/pool/boxed.rs

```diff
@@ -301,6 +305,32 @@ where
 {
 }
 
+#[cfg(feature = "embedded-dma-impl")]
```

Comment:
Should these be removed now when using `StableDeref`?

Reply:
Yes, I removed it.